### DISCUSSION TITLE
No "window.systemColorTheme" setting?

### DISCUSSION THREAD

**Discussion by JohnVeness** [VSCodium/vscodium#1810] [Q&A]
I've just updated to VSCodium 1.86.2 from 1.85.2. Reading the release notes (from Microsoft) I was interested in the https://code.visualstudio.com/updates/v1_86#_new-setting-to-enforce-system-color-theme feature where we can supposedly use a new setting "window.systemColorTheme". However, that setting doesn't seem to appear in my Settings editor. I'm running Linux. Is this a macOS-only setting? I haven't tried in upstream VSCode. Thanks!

**Comment by daiyam** [comment]
No idea... sorry for not being helpful.
